core/src/main/java/com/linecorp/armeria/server/TransientService.java

```diff
@@ -27,4 +30,41 @@
  */
 @FunctionalInterface
 public interface TransientService<I extends Request, O extends Response> extends Service<I, O> {
+
+    /**
+     * Tells whether the specified {@link ActionType} is enabled for this {@link TransientService}.
+     */
+    default boolean countFor(ActionType type) {
+        return false;
+    }
```

Comment:
Should this be dynamic or static? If static, I guess we can just have this:

```java
default Set<ActionType> countedActionTypes() {
    return ALL_ACTION_TYPES;
}
```

.. and expose it in `ServiceConfig` for faster retrieval, because it will be queried very often.